src/sentry/south_migrations/0359_auto__add_index_tagvalue_project_id_key_last_seen.py

```diff
@@ -0,0 +1,955 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
Set this to `True` since I have to kick this off by hand. It'll just prevent it from automatically running during deploy.